.github/workflows/data.yml

```diff
@@ -0,0 +1,51 @@
+name: Download records & Push to repository
+
+on:
+  push:
+    branches: ["main"]
+  schedule:
+    # Run at 8:00 on working days
+    - cron: 0 8 * * 0-5
+
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    if: ${{ !startsWith(github.event.head_commit.message, '[bot]') }}
+    environment:
+      name: download-data
+    env:
+      CI_COMMIT_USER: 'github-actions[bot]'
+      CI_COMMIT_EMAIL: 'github-actions[bot]@users.noreply.github.com'
+      CI_COMMIT_MESSAGE: '[bot] update records'
+      CI_PUSH_REMOTE: git@github.com:$GITHUB_REPOSITORY.git
+      CI_PUSH_BRANCH: 'main'
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
To be clear, for this to work, I will have to add this secret, right?

Reply:
yes. this is using a project-level deploy key. An alternative is to separate the scheduled data CI into another repo and this repo would just get the data from there

Reply:
Great, thanks! I'll set up the secret.

Reply:
I think I can use 
```suggestion
        token: ${{ secrets.NWB_EXTENSIONS_BOT_GITHUB_TOKEN }}
```